recipe/meta.yaml

```diff
@@ -113,8 +130,9 @@ outputs:
       imports:
         - libmambapy
         - libmambapy.bindings
+        - libmambapy.solver
       commands:
-        - python -c "import libmambapy._version; assert libmambapy._version.__version__ == '{{ libmambapy_version }}'"
+        # - python -c "import libmambapy._version; assert libmambapy._version.__version__ == '{{ libmambapy_version }}'"
```

Comment:
Can this test be re-enabled?

Reply:
I will try again but it was giving me some failures at one point.

Reply:
There's no longer an underscore in front of version, so I fixed that and it works now! (: